scripts/build.ps1

```diff
@@ -44,12 +44,6 @@ foreach ($file in [System.IO.Directory]::EnumerateFiles("$PSScriptRoot\..\src",
 }
 
 foreach ($file in [System.IO.Directory]::EnumerateFiles("$PSScriptRoot\..\tests", "*.csproj", "AllDirectories")) {
-    if ($file -match "System.IO.Pipelines.Performance.Tests")
```

Comment:
Test runner does not fail if there is no tests, it would just build and don't running anything. So there is no real need to special case `System.IO.Pipelines.Performance.Tests`.